packages/explorer/bin/explorer.ts

```diff
@@ -1,23 +1,63 @@
 #!/usr/bin/env node
 import { watchFile } from "fs";
 import { readFile } from "fs/promises";
-import minimist from "minimist";
 import path from "path";
 import process from "process";
 import { fileURLToPath } from "url";
+import yargs from "yargs";
 import { ChildProcess, spawn } from "child_process";
 
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 
-const argv = minimist(process.argv.slice(2));
-const port = argv.port || process.env.PORT || 13690;
-const chainId = argv.chainId || process.env.CHAIN_ID || 31337;
-const indexerDatabase = argv.indexerDatabase || process.env.INDEXER_DATABASE || "indexer.db";
-const worldsFile = argv.worldsFile || process.env.WORLDS_FILE || "worlds.json";
-const isDev = !!argv.dev;
-
-let worldAddress = argv.worldAddress || process.env.WORLD_ADDRESS || null;
+// Define command-line options with yargs
+const argv = yargs(process.argv.slice(2))
+  .options({
+    port: {
+      alias: "p",
+      description: "Port number for the server",
+      type: "number",
+      default: process.env.PORT || 13690,
+    },
+    chainId: {
+      alias: "c",
+      description: "Chain ID",
+      type: "number",
+      default: process.env.CHAIN_ID || 31337,
+    },
+    indexerDatabase: {
+      alias: "i",
+      description: "Path to the indexer database",
+      type: "string",
+      default: process.env.INDEXER_DATABASE || "indexer.db",
+    },
+    worldsFile: {
+      alias: "w",
+      description: "Path to the worlds.json file",
+      type: "string",
+      default: process.env.WORLDS_FILE || "worlds.json",
+    },
+    dev: {
+      alias: "d",
+      description: "Run in development mode",
+      type: "boolean",
+      default: false,
+    },
+    worldAddress: {
+      alias: "a",
+      description: "World address",
+      type: "string",
+      default: process.env.WORLD_ADDRESS || null,
+    },
+  })
+  .parseSync();
+
+const port = argv.port;
+const chainId = argv.chainId;
+const indexerDatabase = argv.indexerDatabase;
+const worldsFile = argv.worldsFile;
+const isDev = argv.dev;
```

Comment:
is it necessary to map these to their own variables?

could we either just refer directly to `argv.port` below or explode with `const { port, ... } = yargs(...)`

Reply:
True, now that `yargs` is used here, adjusted